apps/fluent-tester/src/E2E/BootPage/pages/BootTestPage.win.ts

```diff
@@ -1,19 +1,17 @@
-import {
-  HOMEPAGE_BUTTON_BUTTON,
-  HOMEPAGE_CALLOUT_BUTTON,
-  HOMEPAGE_CHECKBOX_BUTTON,
-  HOMEPAGE_FOCUSTRAPZONE_BUTTON,
-  HOMEPAGE_LINK_BUTTON,
-  HOMEPAGE_PERSONA_BUTTON,
-  HOMEPAGE_PERSONACOIN_BUTTON,
-  HOMEPAGE_PRESSABLE_BUTTON,
-  HOMEPAGE_RADIOGROUP_BUTTON,
-  HOMEPAGE_SEPARATOR_BUTTON,
-  HOMEPAGE_SVG_BUTTON,
-  HOMEPAGE_TEXT_BUTTON,
-  HOMEPAGE_THEME_BUTTON
-} from '../../RNTester/Consts';
-import { By } from './BasePage';
+import { HOMEPAGE_CHECKBOX_BUTTON } from '../../../RNTester/TestComponents/Checkbox/consts';
```

Comment:
Are the .win.ts extensions exclusively for win32? Or shared with uwp? If they aren't shared I'd use .win32.ts(x) for win32

Reply:
Everything is going to be shared except for the spec document, which is why I just kept these as ".win", short for windows.